demo/src/app/components/pagination/overview/pagination-overview.component.html

```diff
@@ -0,0 +1,44 @@
+<p>
+  Pagination is a component that only displays page numbers. It will not manipulate your data collection. You will have
+  to split your data collection into pages yourself.
+</p>
+
+
+<ngbd-overview-section [section]="sections['basic-usage']">
+  <p>
+    In order to properly operate a pagination behaviour, there are 3 important notions we have to be familiar with,
+    which are actually available as <code>@Input()</code> on the widget:
+  </p>
+  <ol>
+    <li><code>collectionSize</code> - Number of elements/items in the collection.
+      i.e. the total number of items the
+      pagination should handle.</li>
+    <li><code>pageSize</code> - Number of elements/items per page. i.e. the number
+      of items
+      each page should contain.</li>
+    <li><code>page</code> - The current page.</li>
+  </ol>
+
+  <p>To split the data collection ourselves, we will use <code>*ngFor</code> associated with the <a href="https://angular.io/api/common/SlicePipe"
+      rel="no-opener no-referer"><code>slice</code></a> pipe to extract (or <em>slice</em>) a sub-part of it.</p>
+
+  <p>Corresponding code could be something like:</p>
+  <ngbd-code lang="html" [code]="NGFOR"></ngbd-code>
+  <p>and the associated <code>&lt;ngb-pagination&gt;</code> would be like:</p>
+  <ngbd-code lang="html" [code]="NGB_PAGINATION"></ngbd-code>
+
+  <br />
+  <ngb-alert type="warning" [dismissible]="false">
+    Be aware that both <code>page</code> and <code>pageSize</code> have default values,
+    which are respectively <code>1</code> and <code>10</code>.
+  </ngb-alert>
+
+  <h4>Filtering and sorting</h4>
+  <p>
+    If you want to add filtering or sorting on top of your pagination, we will just have to update the way we split our
```

Comment:
If you want..., **we** will just → If you want, ... **you**'ll just ?

Reply:
done, I picked **you**